doc/nrf/ug_nrf7002.rst

```diff
@@ -0,0 +1,265 @@
++-----------+------------+----------------------+
+| UARTE     | on-chip    | serial               |
++-----------+------------+----------------------+
+| WDT       | on-chip    | watchdog             |
++-----------+------------+----------------------+
+
+Other hardware features are not supported by the |NCS| kernel.
+
+Connections and IOs
+===================
+
+The following are the connections and IOs supported by the development kit.
+
+LED
+---
+
+* LED1 (green) = P1.06
+* LED2 (green) = P1.07
+
+Push buttons
+------------
+
+* BUTTON1 = SW1 = P1.08
+* BUTTON2 = SW2 = P1.09
+* BOOT = SW5 = boot/reset
+
+Wi-Fi control
+-------------
+
+* BUCKEN = P0.12
+* IOVDD CONTROL = P0.31
+* HOST IRQ = P0.23
+* COEX_REQ = P0.28
+* COEX_STATUS0 = P0.30
+* COEX_STATUS1 = P0.29
+* COEX_GRANT = P0.24
+
+Security components
+===================
+
+* Implementation Defined Attribution Unit (`IDAU`_) on the application core.
+
+  The IDAU is implemented with the System Protection Unit and is used to define secure and non-secure memory maps.
+  By default, all of the memory space (Flash, SRAM, and peripheral address space) is defined to be secure accessible only.
+
+* secure boot.
+
+Programming and debugging
+*************************
+
+The nRF5340 application core supports the Armv8-M Security Extension.
+Applications built for the ``nrf7002dk_nrf5340_cpuapp`` board boot by default in the secure state.
+
+The nRF5340 network core does not support the Armv8-M Security Extension.
+nRF5340 IDAU can configure bus accesses by the nRF5340 network core to have secure attribute set.
+This allows to build and run secure-only applications on the nRF5340 SoC.
+
+Building secure/non-secure |NCS| applications with Arm TrustZone
+=================================================================
+
+Applications on the nRF5340 can contain a secure and a non-secure firmware image for the application core.
+You can build the secure image using either |NCS| or `Trusted Firmware M`_ (TF-M).
+You must always build the non-secure firmware images using |NCS|.
+
+.. note::
+   By default, the secure image for the nRF5340 application core is built using TF-M.
+
+Building the firmware with TF-M
+-------------------------------
+
+To build the secure firmware image using TF-M and the non-secure firmware image using |NCS|, follow these steps:
+
+1. Build the non-secure |NCS| application for the application core using ``-DBOARD=nrf7002dk_nrf5340_cpuapp_ns``.
+   To invoke the building of TF-M, the |NCS| build system requires the Kconfig option :kconfig:option:`CONFIG_BUILD_WITH_TFM` to be enabled, which is done by default when building |NCS| as a non-secure application.
+
+   The |NCS| build system performs the following steps automatically:
+
+      a. Build the non-secure firmware image as a regular |NCS| application.
+      #. Build a TF-M (secure) firmware image.
+      #. Merge the output image binaries.
+      #. Optionally, build a bootloader image (MCUboot).
```

Comment:
What about NSIB and using MCUboot as a second-stage bootloader?

Reply:
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/boards/arm/nrf9160dk_nrf9160/doc/index.html#building-the-secure-firmware-with-tf-m I guess we can leave it as is, I found 4 board using exact same steps.

Reply:
These might also not be updated. @kyberdin Could you check this?

Reply:
I'm not sure about the state of NSIB + MCUBoot on 9160, or if we just recommend MCUBoot for it. @sigvartmh any insight here?

Reply:
IMHO, we can resolve this an take it up later when we have more clarity.